api/swagger-spec/v1.json

```diff
@@ -14966,6 +14966,9 @@
    "v1.ConfigMapList": {
     "id": "v1.ConfigMapList",
     "description": "ConfigMapList is a resource containing a list of ConfigMap objects.",
+    "required": [
+     "items"
```

Comment:
I agree with Derek -- and do not see how clients could break from this change, but can easily understand clients breaking from the state of master (ie, omitempty).